src/features/controls.ts

```diff
@@ -290,17 +287,37 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
           // again for this particular <model-element> instance:
           this[$userPromptedOnce] = true;
           this[$waitingToPromptUser] = false;
+          this[$promptElementVisible] = true;
 
           this[$promptElement].classList.add('visible');
         }
       }
 
+      if (this[$promptElementVisible]) {
+        const {
+          left: modelViewerLeft,
+          width: modelViewerWidth,
+        } = this.getBoundingClientRect();
+        const {
+          left: promptLeft,
+          width: promptWidth,
+        } = this[$promptElementSVG].getBoundingClientRect();
+
+        const modelViewerCenter = modelViewerLeft + (modelViewerWidth / 2);
+        const promptCenter = promptLeft + (promptWidth / 2);
+        const promptOffset = promptCenter - modelViewerCenter;
+
+        (this as any)[$scene].pivot.rotation.y = promptOffset * OFFSET_ROTATION_MULTIPLIER;
```

Comment:
It looks like promptOffset is in units of pixels and that it moves by a fixed number of pixels. I wonder if it wouldn't be better to have the motion scale to a % of the canvas size, but keep the angular rotation constant? What I notice now is that the prompt motion relative to the model changes based on how I adjust the window size.

Reply:
Will update to a relative percentage 👍